extensions/wikia/PageShare/tests/PageShareHelperTest.php

```diff
@@ -18,6 +18,7 @@ public function IsValidShareServiceProvider() {
 					'languages:exclude' => [],
 				],
 				'language' => 'en',
+				'isTouchScreen' => 'false',
```

Comment:
You're using everywhere here **string `'false'`**, are you sure? It looks at least strange for me. :frowning: